### PR TITLE
Gnat.ConnectionSupervisor starts up when the application startsup

### PR DESCRIPTION
**Pull request by janitha09**
e.g. iex -S mix

Signed-off-by: Janitha Jayaweera <janitha_j@hotmail.com>

### CODE REVIEW

**Comment by janitha09** [comment]
@autodidaddict @brooksmtownsend

**Review by autodidaddict — COMMENTED** [review]
This looks great I'll check it over tomorrow

File: wadm/lib/config_plan.ex [hunk 1]
```diff
@@ -0,0 +1,11 @@
+defmodule Wadm.ConfigPlan do
+  use Vapor.Planner
+  dotenv()
+
+  config :nats,
+         env([
+           {:host, "NATS_HOST", default: '172.18.0.1'},
```
Comment (autodidaddict) [inline wadm/lib/config_plan.ex:7]: According to a quick google, the use of this address often allows for IP tunneling vulnerabilities. Can we default to `127.0.0.1` instead @janitha09  ?

**Review by autodidaddict — CHANGES_REQUESTED** [review]

**Comment by autodidaddict** [comment]
Everything looks good with the exception of just changing one of the defaults. I will follow this Pr up with one of my own that contains the code to start a LatticeObserver in the appropriate place

File: wadm/lib/config_plan.ex (hunk 1 above)
Comment (janitha09) [reply wadm/lib/config_plan.ex]: fixed

**Review by autodidaddict — APPROVED** [review]

**Review by brooksmtownsend — APPROVED** [review]
This LGTM

**Pull request merged**